tests/gui/models/settings.spec.js

```diff
@@ -9,20 +9,20 @@ describe('Browser: settings', function() {
 
   describe('settings', function() {
 
-    const SUPPORTED_KEYS = _.keys(Store.Defaults.get('settings').toJS());
+    const DEFAULT_KEYS = _.keys(Store.Defaults.get('settings').toJS());
 
     beforeEach(function() {
       this.settings = settings.getAll();
     });
 
     afterEach(function() {
-      _.each(SUPPORTED_KEYS, (supportedKey) => {
+      _.each(DEFAULT_KEYS, (supportedKey) => {
         settings.set(supportedKey, this.settings[supportedKey]);
       });
     });
 
     it('should be able to set and read values', function() {
-      const keyUnderTest = _.first(SUPPORTED_KEYS);
+      const keyUnderTest = _.first(DEFAULT_KEYS);
```

Comment:
Given the number of occurrences of `const keyUnderTest = _.first(DEFAULT_KEYS);`, would it be worth moving it to e.g. just under `const DEFAULT_KEYS = ...` ?